src/pages/neighborhoods/neighborhoods.tsx

```diff
@@ -0,0 +1,51 @@
+import {
+  Box,
+  Circle,
+  Heading,
+  PlusIcon,
+  Square,
+  Wrap,
+  WrapItem,
+} from "@liftedinit/ui";
+import { Neighborhood } from "./neighborhood";
+
+const NEIGHBORHOODS = [
```

Comment:
This will eventually move to the "store" and replace the options in the dropdown.